Source/Core/Core/IOS/Network/KD/NetKDRequest.h

```diff
@@ -126,5 +127,6 @@ class NetKDRequestDevice : public EmulationDevice
   Common::Event m_shutdown_event;
   std::mutex m_scheduler_lock;
   std::thread m_scheduler_timer_thread;
+  std::shared_ptr<NetNCDManageDevice> m_net_device;
```

Comment:
This seems unrelated?

Reply:
Oops, that's from something else I'm working on.